cli/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awarejs-yarn2-sca.yaml-dependency_awarepackage-lock-v3/results.txt

```diff
@@ -0,0 +1,45 @@
+=== command
+SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERSION_CACHE_PATH="<MASKED>" SEMGREP_ENABLE_VERSION_CHECK="0" SEMGREP_SEND_METRICS="off" semgrep --strict --config rules/dependency_aware/js-yarn2-sca.yaml --json targets/dependency_aware/package-lock-v3
+=== end of command
+
+=== exit code
+0
+=== end of exit code
+
+=== stdout - plain
+{
+  "errors": [],
+  "paths": {
+    "_comment": "<add --verbose for a list of skipped paths>",
+    "scanned": [
+      "targets/dependency_aware/package-lock-v3/package-lock.json"
+    ]
+  },
+  "results": [],
```

Comment:
this says no results, so this likely doesn't work yet?

Reply:
well... you would be right.